composer.json

```diff
@@ -31,8 +31,7 @@
 		"phpcompatibility/php-compatibility": "^9.3",
 		"wp-coding-standards/wpcs": "^2.2",
 		"sirbrillig/phpcs-variable-analysis": "^2.8",
-		"wp-phpunit/wp-phpunit": "^5.4",
-		"phpunit/phpunit": "^8.5",
+		"phpunit/phpunit": "^6.5",
```

Comment:
I chose 6.5 here because of the stated requirements for the minimum PHP version. I wanted a version that was compatible with more versions of PHP.